### ISSUE TITLE
403 Forbidden 

### ISSUE THREAD

**Issue by krazyflipj** [nzapponi/homebridge-simplisafe3#12]
Alarm is stuck on disarming. Logs show the below 
![0EA2FF8E-EA2A-459F-90F9-4B51D8BDE1FE](https://user-images.githubusercontent.com/10282165/64468848-47f3fc00-d0de-11e9-9959-cd7e1464b65f.jpeg)

**Comment by nikonratm** [comment]
Assuming if you restart homebridge this happens again after some time? Immediately? What’s your refresh timer set to?

**Comment by krazyflipj** [comment]
> Assuming if you restart homebridge this happens again after some time? Immediately? What’s your refresh timer set to?

happens immediately. Sensor refresh is at 15

**Comment by nikonratm** [comment]
If it happens immediately sounds like a login credentials issue. You’ve double-checked what’s in your config and made sure you can login with those details?

**Comment by krazyflipj** [comment]
> If it happens immediately sounds like a login credentials issue. You’ve double-checked what’s in your config and made sure you can login with those details?

Yes I haven’t changed password and it’s worked before. It’s weird that now I can’t even login to simplisafe. Nothing happens when I hit login

**Comment by krazyflipj** [comment]
Update looks like it’s my vpn. Anyone else run a VPN?

**Comment by nikonratm** [comment]
If you can’t login at all it’s nothing to do with this software, sounds like you gotta reach out to your VPN. Suggest we close this thread.

**Comment by afalahi** [comment]
I'm getting this same issue. I can't access SS on my wifi, but it works using LTE on the app. My door lock was jammed most of the day (dog walker), so I was thinking it was bombarding the API with updates, maybe? IDK, but it seems like my IP is blocked from the SS network. I tried using a VPN, and that works, but I don't want to use a VPN. Anyone got it figured out? I tried calling them, but they never answer the phone!
